javascripts/content_scripts/privly.js

```diff
@@ -690,6 +860,24 @@ var privly = {
     window.addEventListener("message", privly.resizeIframePostedMessage,
```

Comment:
Replace this listener with a generic message listener and dispatch to the proper handler depending on the message.